File: _2482_difference_between_ones_and_zeros_in_row_and_column/main.cpp
class Solution {
public:
    vector<vector<int>> onesMinusZeros(vector<vector<int>>& grid) {
        int m = grid.size();
        int n = grid[0].size();

        vector<int> onesRows(m, 0);
        vector<int> onesCols(n, 0);

        for (int i = 0; i < m; i++) {
            for (int j = 0; j < n; j++) {
                if (grid[i][j] == 1) {
                    onesRows[i] += 1;
                    onesCols[j] += 1;
                }
            }
        }

        for (int i = 0; i < m; i++) {
            for (int j = 0; j < n; j++) {
                grid[i][j] = onesRows[i] + onesCols[j] - (m - onesRows[i]) - (n - onesCols[j]);
            }
        }
        
        return grid;
    }
};

File: _2482_difference_between_ones_and_zeros_in_row_and_column/main.py
class Solution:
    def onesMinusZeros(self, grid: List[List[int]]) -> List[List[int]]:
        m = len(grid)
        n = len(grid[0])

        onesRows = [0] * m
        onesCols = [0] * n

        for i in range(m):
            for j in range(n):
                if grid[i][j] == 1:
                    onesRows[i] += 1
                    onesCols[j] += 1

        for i in range(m):
            for j in range(n):
                grid[i][j] = onesRows[i] + onesCols[j] - (m - onesRows[i]) - (n - onesCols[j])

        return grid